pgxn/neon/unstable_extensions.c

```diff
@@ -0,0 +1,183 @@
+#include <assert.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "postgres.h"
+
+#include "nodes/plannodes.h"
+#include "nodes/parsenodes.h"
+#include "tcop/utility.h"
+#include "utils/errcodes.h"
+#include "utils/guc.h"
+
+#include "unstable_extensions.h"
+
+#if PG_MAJORVERSION_NUM < 16
+#define guc_free(ptr) free(ptr)
+#define guc_malloc(level, size) malloc(size)
+#define guc_strdup(level, s) strdup(s)
```

Comment:
How is this supposed to work for versions < 16 if number of args is different? And I see that tests with these versions failed

It probably makes sense to just use the standard Postgres' pfree, palloc, pstrdup

Reply:
See pgversioncompat changes